src/css/_widget-color.react.css

```diff
@@ -135,21 +135,13 @@
     color: white;
 }
 
-.single div {
+.widget-color-box .single div {
```

Comment:
could we have a single class name like this: `.widget-color-box-single`, `.widget-color-box-active-color`